contrib/babelfishpg_tsql/src/pltsql_coerce.c

```diff
@@ -538,6 +532,10 @@ init_tsql_coerce_hash_tab(PG_FUNCTION_ARGS)
 	/* mark the hash table initialised */
 	inited_ht_tsql_cast_info = true;
 
+    argTypes = (Oid *) palloc(3 * sizeof(Oid));
```

Comment:
can you please explain why is this done?

Reply:
I just added a comment above explaining the reason